bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -104,7 +104,7 @@
         <div class="inner-container">
           <h2>{{ _('Inventing a safer programming language') }}</h2>
           <p>{{ _('Supported by Mozilla, Rust allows browsers, systems and more to run much faster and more safely.') }}</p>
-          <a rel="external" href="https://developer.mozilla.org/docs/Mozilla/Rust" class="button">{{ _('Learn more') }}</a>
+          <a rel="external" href="https://www.youtube.com/watch?list=PLo3w8EB99pqJ74XIGe72c9hBZWz9Y16cY&v=8EPsnf_ZYU0" class="button" data-video-title="Rust and the Future of Systems Programming">{{ _('Watch the video') }}</a>
```

Comment:
Note the string "Watch the video" is already in main.lang, so no l10n tag required.